composer.lock

```diff
@@ -73,7 +73,7 @@
                 "docblock",
                 "parser"
             ],
-            "time": "2015-08-31 12:32:49"
+            "time": "2015-08-31T12:32:49+00:00"
```

Comment:
thus, if people want an updated version of the demo, they should not only update the vendors, but also the demo itself (and so they would get our new lock file and run ``composer install``)